src/arkode/arkode_mristep.c

```diff
@@ -2933,6 +4605,57 @@ int mriStepInnerStepper_Reset(MRIStepInnerStepper stepper, sunrealtype tR,
   }
 }
 
+/* Gets the inner (fast) stepper accumulated error */
```

Comment:
I think these need to be added to the `SUNStepper` interface and `MRIStepInnerStepper_CreateFromSUNStepper` updated so a `SUNStepper` can be used as an `InnerStepper`

Reply:
We could add it when we deprecate or remove `MRIStepInnerStepper` which gives us some more time to adjust the API for analogous SUNStepper functions if needed. In the meantime, `MRIStepInnerStepper_CreateFromSUNStepper` wouldn't be compatible with one type of MRI controller, but I think it would fail graceful with a not implemented error.

Reply:
In another PR I think it would be good to add the three functions needed for H-Tol adaptivity (`GetAccumulatedError`, `ResetAccumulatedError`, and `SetRTol`) since the intent is the have `SUNStepper` replace `MRIStepInnerStepper` and I don't think we should let them get out of sync in capability.